assets/config/0.5.4-coins.json

```diff
@@ -690,6 +690,24 @@
     "active": false,
     "currently_enabled": false
   },
+  "BANANO-BEP20": {
+    "coin": "BANANO-BEP20",
+    "name": "Banano",
+    "nomics_id": "ban-banano",
```

Comment:
https://nomics.com/assets/ban-banano

Reply:
https://nomics.com/assets/ban-banano/widget shows
`<div class="nomics-ticker-widget" data-name="Banano" data-base="BAN" data-quote="USD"></div><script src="https://widget.nomics.com/embed.js"></script>`
so `nomics_id` is `BAN`
on nomics, the ids are not in the URL

Reply:
k, will review and repair in seperate PR